src/client.js

```diff
@@ -112,11 +98,7 @@ export function execContainer(system, id) {
         Cmd: ["/bin/sh"],
     };
 
-    return new Promise((resolve, reject) => {
-        podmanCall("libpod/containers/" + id + "/exec", "POST", {}, system, JSON.stringify(args))
-                .then(reply => resolve(JSON.parse(reply)))
-                .catch(reject);
-    });
+    return podmanJson("libpod/containers/" + id + "/exec", "POST", {}, system, JSON.stringify(args));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1339-20230706-033702-672c013e-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)